manager/allocator/networkallocator/networkallocator_test.go

```diff
@@ -753,6 +753,10 @@ func (a *mockIpam) DiscoverDelete(dType discoverapi.DiscoveryType, data interfac
 	return nil
 }
 
+func (a *mockIpam) IsBuiltIn() bool {
```

Comment:
This is coming from this PR: https://github.com/docker/swarmkit/commit/8afb5ec9d7e27c9b607a58aa81b7eb44e7c9d6c0